scripts/mariadb/create_stats_schema.sql

```diff
@@ -119,4 +130,189 @@ CREATE TABLE
 		Hits MEDIUMINT UNSIGNED NOT NULL,
 		PRIMARY KEY (WeaponStatsID),
 		CONSTRAINT `fk_class_stats` FOREIGN KEY (ClassStatsID) REFERENCES ClassStats (ClassStatsID) ON DELETE CASCADE ON UPDATE RESTRICT
-	) ENGINE = InnoDB;
\ No newline at end of file
+	) ENGINE = InnoDB;
+
+CREATE TABLE
+	IF NOT EXISTS OverallStatsAllTime (
+		SteamID BIGINT UNSIGNED NOT NULL,
+		PlayerName VARCHAR(32) CHARACTER,
+		NumberOfGames SMALLINT UNSIGNED NOT NULL,
+		AverageDPM FLOAT NOT NULL,
+		AverageKills FLOAT NOT NULL,
+		AverageAssists FLOAT NOT NULL,
+		AverageDeaths FLOAT NOT NULL,
+		AverageAirshots FLOAT NOT NULL,
+		AverageHeadshots FLOAT NOT NULL
+		SET
+			utf8mb4 COLLATE utf8mb4_unicode_ci,
+			PRIMARY KEY (SteamID)	
+	)
+
+CREATE TABLE
+	IF NOT EXISTS OverallStatsRecent (
+		SteamID BIGINT UNSIGNED NOT NULL,
+		PlayerName VARCHAR(32) CHARACTER,
+		NumberOfGames SMALLINT UNSIGNED NOT NULL,
+		AverageDPM FLOAT NOT NULL,
```

Comment:
Maybe these fields should be nullable?
What if you aggregate stats for people who only play medic/ never played sniper etc things like that

Reply:
another thought:
So when you click into a player - we see all time records and stuff for the player.
i think all these tables can also include those 'personal bests'.

we don't always select all that data - since it would blow up the overview pages - but we can select all that stuff from this table on the 'player's best' pages

Reply:
so like highest airshots, highest dmg, most drops lol and so on